geotrek/common/parsers.py

```diff
@@ -65,6 +65,9 @@ class DownloadImportError(ImportError):
 
 
 class Parser:
+    """
+    provider: Allow to differentiate multiple GeotrekParser for the same model
```

Comment:
'provider' is not only used for GeotrekParser, isn't it ?
It can be used for any parser, I think (Apidae, Toursinsoft...)